apps/src/weblab2/Weblab2View.tsx

```diff
@@ -208,8 +204,6 @@ const Weblab2View = () => {
               if (configName === 'project') {
                 setSource(newConfig as ProjectType);
               } else if (configName === 'config' || configName === 'layout') {
-                (newConfig as ConfigType).EditorComponent =
```

Comment:
@thomasoniii is it ok that I removed this since we don't support providing an editor component anymore?

Reply:
Oh. That's a shame. I thought the flexibility would've been nice.

Anyway, sure, if not supported can be safely tossed.